.readthedocs.yaml

```diff
@@ -18,9 +18,13 @@ build:
   apt_packages:
     - "graphviz"
   os: ubuntu-22.04
+  env:
+    READTHEDOCS_OUTPUT: "build"  # Set the environment variable
```

Comment:
Could you please explain what this is for? It looks like it breaks the build: 
https://readthedocs.org/projects/pybamm/builds/26675902/

Reply:
This variable is  intended to define the output directory for build artifacts like link checks or PDFs.